Tools/autotest/arduplane.py

```diff
@@ -3291,7 +3291,23 @@ def fly_landing_baro_drift(self):
         self.wait_ready_to_arm()
         self.arm_vehicle()
 
-        self.fly_mission("ap-circuit.txt", mission_timeout=1200)
+        # Load and start mission
+        self.load_mission("ap-circuit.txt", strict=True)-1
+        self.set_current_waypoint(0, check_afterwards=False)
```

Comment:
Why not checking afterwards?

Reply:
I actually copied the lines from fly_mission, I can try adding the check